### ISSUE TITLE
Publish update to a11y statement

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1794]
## Description
As a user, I need to be able to do understand what kind of a11y initiatives have been undertaken so that I can provide Notify to my team.
As a user, I need to know what are the know a11y bugs, so that I am prepared to use Notify with assitive technology. 
As a public servant using assistive technology, I need to be able to assess accessibility issues so that I know whether I will be able to use Notify in my work without difficulty.

WHY are we building?
- We need to keep our a11y statement up to date with our latest efforts and knowledge
- Keep our A11y statement accessible and current

WHAT are we building?
- Updates to the statement following an a11y audit
- Next iteration of our A11y statement should
(1) update known issues and remove fixed issues
Also after removing "Learn more", the "For security reasons" section may be a bit awkward so perhaps we can revisit that in next iteration.
(2) Pete says we can improve the accessibility of the statement itself by moving the table columns that are headings into a element. Also it’s not possible to insert bullets into the details dropdown, so we used dashes as work-around but would be good to find a better solution

VALUE created by our solution
- Accurate and up to date info.
- Our product is more accessible for our users

## Documentation and Artifacts
Google docs, figma mockups, ADRs, screenshots etc.

## Acceptance Criteria

* A11y
* Bilingualism
* Measuring success and metrics

## GC Articles Publish checklist
- [x]  Supporting feature is released and unflagged
- [x]  Create or update slug
- [x]  Page content and slug is translated [Localization: A11y statement updates #352](https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/zh/352)
- [x]  Document links to and from the page
- [x]  Add page slug and title to [inventory](https://docs.google.com/document/d/1N6B4fSwoM40m-pDLI70BvLfL7O0n04ndyAXYaIwNbNw/edit)
- [x]  If new page, add WAF rule
- [x]  Create new page or update existing page content
- [x]  Create or update translation to the page
- [x]  Link translations by choosing an assigned translation
- [x]  If new page, save as draft
- [x]   if new page, ensure this is added to our UI test suite
- [x]  If existing page, update the page
- [x]  Post page in Slack for last minute reviews
- [x]  On staging: Clear the cache through GC Notify's admin panel
- [x]  On staging: Check links, images and language toggle
- [x]  Give a heads up to the app-team channel once ready to publish
- [x]  On staging and production: Clear the cache through GC Notify's admin panel

**Comment by schancase** [comment]
adding blocked tag to show still waiting on vendor

**Comment by schancase** [comment]
hi @YedidaZalik putting your name on this feature :-) cc @connorscarolyns

**Comment by YedidaZalik** [comment]
Currently updating this page and closing this card as this work is being bundled into A11y feedback flow work

**Comment by YedidaZalik** [comment]
@connorscarolyns this is not done yet

**Comment by schancase** [comment]
yes I just noticed this as well. I am re-opening this feature as there is still a child task in progress. cc @YedidaZalik @connorscarolyns

**Comment by schancase** [comment]
removing sprint goal tag only because it is still open but was not identified as a sprint goal for this sprint (this helps my calculations be accurate when doing end of sprint workflow metrics)

**Comment by YedidaZalik** [comment]
All tasks are done, closing now
